.claude/ralph-loop.local.md

```
@@ -0,0 +1,78 @@
+
+```bash
+gh issue edit <issue> --add-assignee @me
+git fetch origin
+git checkout -b feat/<desc>-<issue> origin/main
+```
+
+### Planning
+
+Before implementing, use the **Plan agent** to create an implementation plan:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)